src/helpers/transaction.ts

```diff
@@ -1,16 +1,14 @@
-import { addHexPrefix, toChecksumAddress } from 'ethereumjs-util';
-import { TransactionReceipt, TransactionResponse } from 'ethers/providers';
+import { BigNumber } from '@ethersproject/bignumber';
+import { BytesLike, hexlify } from '@ethersproject/bytes';
+import { TransactionReceipt, TransactionResponse } from '@ethersproject/providers';
 import {
-  Arrayish,
-  BigNumber,
-  bigNumberify,
-  formatEther,
-  hexlify,
-  parseTransaction,
+  parse as parseTransaction,
   recoverAddress,
-  serializeTransaction,
+  serialize as serializeTransaction,
   Transaction
-} from 'ethers/utils';
+} from '@ethersproject/transactions';
+import { formatEther } from '@ethersproject/units';
+import { addHexPrefix, toChecksumAddress } from 'ethereumjs-util';
```

Comment:
We can use ethers.js here instead of ethereumjs.

Reply:
What are these functions called in `ethers.js`?

Reply:
* [`getAddress`](https://github.com/ethers-io/ethers.js/blob/master/packages/address/src.ts/index.ts#L77) in `@ethersproject/address`
* Couldn't find a proper replacement for `addHexPrefix` but if we want to remove `ethereumjs-util` fully at some point, it should be easy enough to make a util function for that.